commands/update.bee.inc

```diff
@@ -123,9 +149,47 @@ function update_db_bee_callback() {
  * @todo  Consider a way to download first, check the download has worked before delete.
  *
  */
-function update_bee_callback($arguments) {
+function update_bee_callback($arguments, $options) {
   global $_bee_backdrop_root;
   require_once $_bee_backdrop_root . '/core/includes/file.inc';
+  $data = update_bee_gather_available_updates($arguments, $options);
+  if ($data != NULL) {
+    update_bee_render_table_output($data);
+    // Prompt to continue.
+    if (!bee_confirm(bt('Would you like to continue?'), FALSE)) {
+      return;
+    }
+    foreach ($data as $item) {
+      $folder = backdrop_get_path($item['project_type'], $item['name']);
+      if (is_dir($folder)) {
+         bee_delete($folder);
+      }
+      download_bee_callback(array('projects' => array($item['name']),), []);
+      echo '';
+    }
+  } else {
+    bee_message(bt('No Modules or Themes to Update'));
+  }
+}
+
+/**
+ * Command callback: Show module and theme updates.
+ */
+function update_list_bee_callback($arguments, $options) {
+  $data = update_bee_gather_available_updates($arguments, $options);
+  $is_list_format = isset($options['format']) && $options['format'] === 'list';
+  if ($data === NULL && !$is_list_format) {
+    bee_message(bt('No Modules or Themes to Update'));
+    return;
+  }
+  if ($is_list_format) {
+    update_bee_render_list_output($data);
+  } else {
+    update_bee_render_table_output($data);
+  }
+}
+
+function update_bee_gather_available_updates($arguments, $options) {
```

Comment:
Need a function doc for all functions including helper functions.  For helper functions we tend to use the verb 'get' rather than 'gather', so for consistency, please change to that.